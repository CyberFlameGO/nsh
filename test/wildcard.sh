cd test/playground
echo *
echo */*
echo */*/*.mp3
echo */*/smelly*.mp3
echo monty_*/spam_*.txt
echo fri?nds
echo friend?/*
echo friends/*
