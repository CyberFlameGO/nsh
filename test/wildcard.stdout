README.md
friends
monty_python
friends/songs
friends/the_one_with_the_lottery.txt
monty_python/cheese.txt
monty_python/parrot.txt
monty_python/spam_egg_spam_spam_bacon_and_span.txt
monty_python/spam_spam_spam_spam_spam_spam_baked_beans_spam_spam_spam_and_spam.txt
friends/songs/ross_can.mp3
friends/songs/smelly_cat.mp3
friends/songs/smelly_cat.mp3
monty_python/spam_egg_spam_spam_bacon_and_span.txt
monty_python/spam_spam_spam_spam_spam_spam_baked_beans_spam_spam_spam_and_spam.txt
friends
friends/songs
friends/the_one_with_the_lottery.txt
friends/songs
friends/the_one_with_the_lottery.txt
